assignments/prototypes.js

```diff
@@ -14,6 +14,15 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
+// GameObject Constructor
+function GameObject(attributes) {
+  this.createdAt = attributes.createdAt;
+  this.dimensions = attributes.dimensions; 
+}// GameObject
+
+GameObject.prototype.destroy = function() {
+  console.log('Object was removed from the game.');
```

Comment:
If you look at the tests down below, they are running a console log on the `.destroy()` method. You never want to create a situation where you are console.logging a console.log, because that can lead to some strange behavior. I think you would be better off with a return here, or to alter the tests slightly and remove the console log from the test